src/app/services/rendering-container-image.service.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable } from "@angular/core";
+import { log } from "@batch-flask/utils";
+import {
+    RenderApplication, RenderEngine,
+    RenderingContainerImage, RenderingImageReference,
+} from "app/models/rendering-container-image";
+import { Observable } from "rxjs";
+import { map, publishReplay, refCount, share } from "rxjs/operators";
+import { GithubDataService } from "./github-data";
+
+const dataFile = "data/rendering-container-images.json";
+
+@Injectable()
+export class RenderingContainerImageService {
+
+    public imageReferences: Observable<RenderingImageReference[]>;
+    public containerImages: Observable<RenderingContainerImage[]>;
+
+    constructor(
+        private githubDataService: GithubDataService) {
+
+        const data = this.githubDataService.get(dataFile).pipe(
+            map((content) => {
+                try {
+                    return JSON.parse(content);
+                } catch (error) {
+                    log.error(`File is not valid json: ${error.message}`);
+                    return { image_references: [], container_images: [] };
+                }
+            }),
+            publishReplay(1),
+            refCount());
+
+        this.imageReferences = data.pipe(map(x => x.image_references));
```

Comment:
I would try to change to camelCase for the json properties

Reply:
done